packages/trail-core/lib/index.js

```diff
@@ -87,16 +88,36 @@ class TrailsManager {
           ("when" >= ${from.toISO()} AND "when" <= ${to.toISO()})
     `
 
+    // Prepare the query to get Count
+    const sqlCount = SQL`
+      SELECT
+          count(*)
+        FROM trails
+        WHERE
+          ("when" >= ${from.toISO()} AND "when" <= ${to.toISO()})
```

Comment:
minor: you could unconditionally append this to both sql and sqlCount so you avoid the duplication in the definition of the query